examples/skr/aci/README.md

```diff
@@ -18,7 +18,9 @@ Deploying a confidential container group requires generating a security policy t
 
 The ARM template can be used directly to generate a security policy. The following command generates a security policy and automatically injects it into the template. Make sure `--debug-mode` option is included so that the generated policy allows shelling into container to see the released key in this example. NOTE: the current image used in the ARM template is built upon commit id a82b530.
 
-```
+The ARM template can be used directly to generate a security policy. The following command generates a security policy and automatically injects it into the template. Make sure `--debug-mode` option is included so that the generated policy allows shelling into container to see the released key in this example. NOTE: the current image used in the ARM template is built upon commit id a82b530.

```

Comment:
I don't feel mentioning the commit id is necessary. Does the user have any practical use for the commit id?